src/EnergyPlus/HeatRecovery.hh

```diff
@@ -680,63 +470,34 @@ namespace HeatRecovery {
 
 } // namespace HeatRecovery
 
+struct ErrorTracker2
+{
+    std::string OutputChar;          // character string for warning messages
+    std::string OutputCharLo;        // character string for warning messages
+    std::string OutputCharHi;        // character string for warning messages
+    std::string CharValue;           // character string for warning messages
+    Real64 TimeStepSysLast = 0.0;    // last system time step (used to check for downshifting)
+    Real64 CurrentEndTime = 0.0;     // end time of time step for current simulation time step
+    Real64 CurrentEndTimeLast = 0.0; // end time of time step for last simulation time step
+};
```

Comment:
The state data here also had a ton of repeated variable types for more error tracking.  I could see some future work to clean this up further, but for now, I collected them into a small struct to at least make it one step better.